go/teams/implicit.go

```diff
@@ -263,6 +280,13 @@ func LookupOrCreateImplicitTeam(ctx context.Context, g *libkb.GlobalContext, dis
 		return res, teamName, impTeamName, err
 	}
 
+	if err := assertIsDisplayNameNormalized(lookupName); err != nil {
+		// Do not allow display names with mixed letter case - while it's legal
+		// to create them, it will not be possible to load them because API
+		// server always downcases during normalization.
```

Comment:
Does it have to be lowercase for looking up, or just creating?

Reply:
for both, api endpoints will just refuse to find that team, at least from my testing. maybe there's something else at play but I've exhausted time I had to debug this :(

this at least stops you from creating "broken" imp teams.

Reply:
well this works for example `keybase chat send "ZAPU.NET@https" test`, or do you just mean for unresolved impteams?

Reply:
this resolves to a username so it's fine. but something like `ZAPU.NET.PL@https` would make a new implicit team which would sort of work in chat but cause different errors along the way - i think including during sbs resolution handler.

Reply:
@maxtaco can you take a look at this change? it prevents creating an implicit team that client will not be able to load by name. I don't know what the real fix is here yet

Reply:
I'm fine with this but I don't understand why we don't just downcase all the time.